apps/designer/app/designer/features/breakpoints/use-next-breakpoint.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from "react";
+import { useSelectedBreakpoint } from "~/designer/shared/nano-states";
+import { useBreakpoints } from "~/shared/nano-states";
+import { utils } from "@webstudio-is/project";
+
+/**
+ * Return the next breakpoint from the currently selected one, sorted by the `sort()`
+ */
+export const useNextBreakpoint = () => {
+  const [selectedBreakpoint] = useSelectedBreakpoint();
+  const [breakpoints] = useBreakpoints();
+
+  const sortedBreakpoints = useMemo(
+    () => utils.breakpoints.sort(breakpoints),
+    [breakpoints]
+  );
+
+  return useMemo(() => {
+    if (selectedBreakpoint === undefined) {
+      return;
```

Comment:
Return undefined explicitly?

Reply:
we don't need to do that, why?

Reply:
return; is a equally good, same thing, quite clear etc

Reply:
actually a benefit over other values, since you don't have to type anything and I find it very explicit